adlfs/multithread.py

```diff
@@ -135,7 +135,7 @@ def touch(self, src, dst):
             with open(dst, 'wb'):
                 pass
 
-        self.client.run(nthreads, monitor, before_scatter=touch)
+        self.client.run(nthreads, monitor, before_start=touch)
```

Comment:
It should be called, else any `info` to check the file size might get not-found or else previous size.